### DISCUSSION TITLE
02230290

### DISCUSSION THREAD

**Discussion by Namgay282004** [kevinlin1/just-the-class#38] [Q&A]
https://www.figma.com/file/M89KmhuuxVgkJBQcb373N1/Untitled?type=design&node-id=0-1&mode=design&t=YWSHfrvpNNNrinY2-0

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
